src/router/index.js

```diff
@@ -1,55 +1,152 @@
 import Vue from "vue";
 import Router from "vue-router";
 import Movie from "../views/Movie.vue";
+import Home from "../views/Home.vue";
 import Login from "../views/Login.vue";
 import Register from "../views/Register.vue";
 import Search from "../views/Search.vue";
 import About from "../views/About.vue";
 import Contact from "../views/Contact.vue";
 import MovieDetail from "../views/MovieDetail.vue";
+import WatchMovie from "../views/WatchMovie.vue";
+import MovieCMS from "../views/MovieCMS.vue";
+import VueCookies from "vue-cookies";
 
 Vue.use(Router);
 const routes = [
   {
     path: "/",
+    name: "Home",
+    component: Home,
+    beforeEnter: (to, from, next) => {
+      if (!VueCookies.get("Account")) next();
+      else if (VueCookies.get("Account").role == "guest") {
+        next({ name: "Movie" });
+      } else {
+        next({ name: "MovieCMS" });
+      }
+    },
+  },
+  {
+    path: "/movie",
```

Comment:
Nào nào. Chuẩn đặt url đâu roài.

Reply:
anh ơi, cái này là em dùng be của project ngoài TT. Các bạn ấy viết thế huhu. Giờ phải sửa lại hết ạ.

Reply:
ựa. Vâng ạ =)))